### PR TITLE
✨ Implement Bitexbook ticker

### LINKED ISSUE #439: Implement Bitexbook Ticker, Orderbook, Trades
website: https://bitexbook.com

### PR DESCRIPTION
**Pull request by Swingcloud**
- What is the purpose of this Pull Request? as title
- What is the related issue for this Pull Request (if this PR fixes issue, prepend with "Fixes" or "Closes")? closes #439
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

ticker about `btc_usd`
```ruby
{
 "symbol": "btcusd",
 "alias": "BTC/USD",
 "currency_base": "btc",
 "currency_base_name": "Bitcoin",
 "currency_quoted": "usd",
 "currency_quoted_name": "US Dollar",
 "statistic": [
  {
   "period": "last_24_hours",
   "price_open": 7119.7211,
   "price_close": 7039.0203,
   "price_low": 6710.6366,
   "price_high": 7494.6588,
   "price_bid": 7000,
   "price_ask": 7499.999,
   "volume_quoted": 748.4065118094067,
   "volume_base": 0.10544345000000004,
   "orders_buy": 45,
   "orders_sell": 35
  }
 ]
}
```
use `volume_base` as volume 
use server time as timestamp 
```ruby
"meta": {
 "from": "2018-10-24T03:14:29.863472757Z",
 "to": "2018-10-25T03:14:29.863473275Z",
 "server_time": "2018-10-25T03:14:29.863473507Z"
}
```

### CODE REVIEW

File: lib/cryptoexchange/exchanges/bitexbook/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Bitexbook::Market::API_URL}/symbols/statistic?last_day=true"
```
Comment (wongy91) [inline lib/cryptoexchange/exchanges/bitexbook/services/market.rb]: the ticker returned is 24h? do they have minutely (or better frequency) data endpoint?

File: lib/cryptoexchange/exchanges/bitexbook/services/market.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        end
+
+        def adapt_all(output)
+          timestamp = HashHelper.dig(output, 'meta', 'server_time')
```
Comment (wongy91) [inline lib/cryptoexchange/exchanges/bitexbook/services/market.rb]: `server_time` changes but not the price. In this case, leave it blank.

Comment (Swingcloud) [reply lib/cryptoexchange/exchanges/bitexbook/services/market.rb]: if we use minutely response then maybe we can use that timestamp

File: lib/cryptoexchange/exchanges/bitexbook/services/market.rb (hunk 1 above)
Comment (Swingcloud) [reply lib/cryptoexchange/exchanges/bitexbook/services/market.rb]: according to their api doc that have minutely like this:
https://api.bitexbook.com/api/v2/symbols/statistic?grouping=1
maybe we can grab the latest one 
but volume needs to refer to 1 day?

Comment (wongy91) [reply lib/cryptoexchange/exchanges/bitexbook/services/market.rb]: that is one way but that's 2 calls for 1 ticker. Wondering if they have better endpoint that we can use.

**Comment by cvnghv** [comment]
Hello! 
Is there any news? We can improve our API without any problems. Just tell us what you need to be changed and we will do it in the near future!
Looking forward to your reply!

**Comment by Swingcloud** [comment]
Hello @cvnghv, I think we need an endpoint that can have minutely update about the price (price_close, price_low, price_high, price_bid, price_ask)  but also has 24hr volume, right now if we need to do that we need to make two API call to grab that info.
So maybe can you provide a new endpoint that we can grab all info with just one API call?

**Comment by cvnghv** [comment]
@Swingcloud will do it as soon as possible. Thanks!

**Comment by cvnghv** [comment]
Hello @Swingcloud, we have added volume_24h_quoted and volume_24h_base, which contain daily volume. Please, check it: https://api.bitexbook.com/api/v2/symbols/statistic?grouping=1

**Comment by cvnghv** [comment]
Hello. Any updates?
Feel free to ask if something is wrong with API.

**Pull request closed without merging**